haystack/utils.py

```diff
@@ -49,6 +82,27 @@ def print_answers(results: dict, details: str = "all"):
         pp.pprint(results)
 
 
+def print_documents(results: dict, max_text_len: int=None):
+    print(f"Query: {results['query']}")
+    pp = pprint.PrettyPrinter(indent=4)
+    for d in results["documents"]:
+        print()
+        new_text = d["text"][:max_text_len]
+        if len(new_text) != len(d["text"]):
+            new_text += "..."
+        results = {
+            "name": d["meta"]["name"],
+            "text": new_text
+        }
+        pp.pprint(results)
+
+
+def print_answers_gen(results: dict):
```

Comment:
I assume this is for generated answers? Can't we adjust `print_answers()` or adjust the result format from the generator so that we need a special method here? I think we should try to "standardize" the formats more in the next weeks. The more special helpers we add now on top, the more we'll have to refactor again soon...

Reply:
I have merged this fn with `print_answers()` in a rather hacky way. Anticipating that Reader and Generator outputs will be standardized and that we can come up with a proper solution once we have decided on that format.